clouddrift/sphere.py

```diff
@@ -435,3 +435,133 @@ def sphere_to_plane(
     y[..., :] = np.cumsum(dy, axis=-1)
 
     return x, y
+
+
+def earthsphere_to_cartesian(
+    lon: np.ndarray,
+    lat: np.ndarray,
+    earth_radius: Optional[float] = EARTH_RADIUS_METERS,
```

Comment:
Same here, can be just `radius` that defaults to `EARTH_RADIUS_METERS`.

Reply:
ok!